docs/partial-inputs.ipynb

```diff
@@ -0,0 +1,446 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "a1886f5a-8669-40e7-8fae-e45619570e3c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Found 12 items\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Point over Monchique Portugal\n",
+    "poi = 37.30939, -8.57207\n",
+    "\n",
+    "# Dates of a large forest fire\n",
+    "start = \"2018-07-01\"\n",
+    "end = \"2018-09-01\"\n",
+    "\n",
+    "catalog = pystac_client.Client.open(STAC_API)\n",
+    "\n",
+    "search = catalog.search(\n",
+    "    collections=[COLLECTION],\n",
+    "    datetime=f\"{start}/{end}\",\n",
+    "    bbox=(poi[1] - 1e-5, poi[0] - 1e-5, poi[1] + 1e-5, poi[0] + 1e-5),\n",
+    "    max_items=100,\n",
+    "    query={\"eo:cloud_cover\": {\"lt\": 80}},\n",
+    ")\n",
+    "\n",
+    "items = search.get_all_items()\n",
+    "\n",
+    "print(f\"Found {len(items)} items\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c4ba5c36-90a6-427c-80c5-2a83ad11a1b0",
+   "metadata": {},
+   "source": [
+    "## Download the data\n",
+    "Get the data into a numpy array and visualize the imagery. The burn scar is visible in the last five images."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c371501c-3ef0-4507-9073-0521a1c733be",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Extract coordinate system from first item\n",
+    "epsg = items[0].properties[\"proj:epsg\"]\n",
+    "\n",
+    "# Convert point into the image projection\n",
+    "poidf = gpd.GeoDataFrame(\n",
+    "    pd.DataFrame(),\n",
+    "    crs=\"EPSG:4326\",\n",
+    "    geometry=[Point(poi[1], poi[0])],\n",
+    ").to_crs(epsg)\n",
+    "\n",
+    "coords = poidf.iloc[0].geometry.coords[0]\n",
+    "\n",
+    "# Create bounds of the correct size, the model\n",
+    "# requires 512x512 pixels at 10m resolution.\n",
+    "bounds = (\n",
+    "    coords[0] - 2560,\n",
+    "    coords[1] - 2560,\n",
+    "    coords[0] + 2560,\n",
+    "    coords[1] + 2560,\n",
+    ")\n",
+    "\n",
+    "# Retrieve the pixel values, for the bounding box in\n",
+    "# the target projection. In this example we use only\n",
+    "# the RGB and NIR band groups.\n",
+    "stack = stackstac.stack(\n",
+    "    items,\n",
+    "    bounds=bounds,\n",
+    "    snap_bounds=False,\n",
+    "    epsg=epsg,\n",
+    "    resolution=10,\n",
+    "    dtype=\"float32\",\n",
+    "    rescale=False,\n",
+    "    fill_value=0,\n",
+    "    assets=BAND_GROUPS[\"rgb\"] + BAND_GROUPS[\"nir\"],\n",
+    "    resampling=Resampling.bilinear,\n",
+    ")\n",
+    "\n",
+    "stack = stack.compute()\n",
+    "\n",
+    "stack.sel(band=[\"red\", \"green\", \"blue\"]).plot.imshow(\n",
+    "    row=\"time\", rgb=\"band\", vmin=0, vmax=2000, col_wrap=7\n",
```

Comment:
Since there are 12 images, could maybe use col_wrap=6? Or did you want the top row to be pre-fire, and bottom row to be post-fire images?

```suggestion
    "stack.sel(band=[\"red\", \"green\", \"blue\"]).plot.imshow(\n",
    "    row=\"time\", rgb=\"band\", vmin=0, vmax=2000, col_wrap=6\n",
```